app/controllers/resources_controller.rb

```diff
@@ -10,7 +10,7 @@ def create
     begin
       response_cocina_obj = Dor::Services::Client.objects.register(params: cocina_model)
     rescue Dor::Services::Client::UnexpectedResponse => e
-      return render build_error('Bad Request', e, '400', :bad_request)
+      return render build_error('Error registering object with dor-services-app', e, '502', :bad_gateway)
```

Comment:
is this only called for registering objects?  (is the language correct)

Reply:
Yes, given where the rescue is placed this is only called for registering objects.